README.md

```diff
@@ -166,56 +166,209 @@ When using sidecar mode the OpenTelemetry collector container will have the envi
 
 ### OpenTelemetry auto-instrumentation injection
 
-The operator can inject and configure OpenTelemetry auto-instrumentation libraries. Currently Java, NodeJS and Python are supported.
+The operator can inject and configure OpenTelemetry auto-instrumentation libraries.
+Currently Java, NodeJS and Python are supported. Each language auto-instrumentation supports different exporters.
+Auto-instrumentation is configured by [Instrumentation](./apis/v1alpha1/instrumentation_types.go) resource.
 
-To use auto-instrumentation, configure an `Instrumentation` resource with the configuration for the SDK and instrumentation.
+#### Instrumentation resource
+
+OpenTelemetry-Operator uses Instrumentation resource to define auto-instrumentation configuration.
+A single `Instrumentation` resource can host configuration to all available auto-instrumentations.
+It is mandatory to create an `Instrumentation` resource.
 
 ```yaml
-kubectl apply -f - <<EOF
 apiVersion: opentelemetry.io/v1alpha1
 kind: Instrumentation
 metadata:
+  ## name - defines name of the instrumentation resource
   name: my-instrumentation
 spec:
+  ## env - a list of Env objects which will be added to the instrumented pod
+  env: 
+    - name: MY_NAMESPACE
+      valueFrom:
+        fieldRef:
+          fieldPath: metadata.namespace
```

Comment:
This is not a great example, the namespace should be injected to the resource automatically.

I would like to avoid people using spec.env to configure their apps (just keep in mind)